packages/metal-clay-button/src/MetalClayButton.soy

```diff
@@ -15,13 +16,14 @@
 	{@param? type: string}
 	{@param? value: string}
 
+
 	{if $href}
-		<a href="{$href}" {call .attributes data="all" /}>
+		<a href="{$href}" {call .attributes data="all" /} aria-label="{$aria_description}">
```

Comment:
I think it's better we have `aria_label` attribute instead of `aria_description` to match with the aria property name.